runestone/parsons/js/parsons.js

```diff
@@ -2250,12 +2253,13 @@ export default class Parsons extends RunestoneBase {
     //    checkCount = how many times it has been checked before correct
     //    userRating = 0..100 how good the person is at solving problems
     initializeAdaptive() {
-        this.adaptiveId = eBookConfig.email;
-        if (this.adaptiveId == undefined) {
-            this.adaptiveId = "parsons";
-        } else {
-            this.adaptiveId = this.adaptiveId + "-parsons";
-        }
+        // this.adaptiveId = eBookConfig.email;
```

Comment:
When the unit tests pass, you will need to remove these lines before I accept the PR.   When you are using git there is no reason to leave commented out code lying around as you can always check out previous versions of the code.  Thanks.

Reply:
Sure I will remove it. Sorry for the incovenience

Reply:
No worries, this is what the review process is for.